djangocms_versioning_filer/cms_config.py

```diff
@@ -78,3 +78,6 @@ def file_versionable():
 class FilerVersioningCMSConfig(CMSAppConfig):
     djangocms_versioning_enabled = True
     versioning = list(versioning_filer_models_config())
+    djangocms_moderation_enabled = True
+    moderated_models = [File]
```

Comment:
please provide list of filer models, not only `File`. Do something like
```
for model_name in filer.settings.FILER_FILE_MODELS:
         model = apps.get_model(model_name)
```
for more deatils see above generator (lines 54-69)